setup-custom-repository.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -x
+usage() {
+  echo <<INFO "$(basename "${0}")" --repository REPOSITORY --repository-key KEY
+    --repository REPOSITORY path to the MaxScale repository
+    --repository-key KEY               URL to the repository key
+INFO
+}
+
+repository=
+key=
+
+while [[ ${1:-} ]]; do
+  case "$1" in
+    "--repository")
+      shift
+      repository="$1"
+      shift
+      ;;
+    "--repository-key")
+      shift
+      key=$1
+      shift
+      ;;
+    "--help")
+      usage
+      exit
+      ;;
+    *)
+      echo "unknown option $1"
+      exit 1
+      ;;
+  esac
+done
+
+if [[ -z $repository ]]; then
+  echo "Please specify path to the repository via --repository"
+  exit 1
+fi
+
+if [[ -z $key ]]; then
+  echo "Please specify the URL to the repository key via --key"
+  exit 1
+fi
+
+echo 'Adding repository signing key'
+if apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "${key}"
```

Comment:
this is executed only in case of creation CI Docker image which does not happen in docker hub CI. My idea is: if we build production images for Docker hub (or docker hub CI does it) `docker build` is executed without any arguments and `etup-custom-repository.sh` is not executed.
REPOSITORY and REPOSITORY_KEY arguments are defined only in my CI when I build CI version of server and docker images 

in case of production build,` mariadb_repo_setup` takes care of key importing